build/script/make.build

```diff
@@ -73,5 +78,11 @@ for binname in ${binlist}
 do
   makebuild ${binname} bin
 done
+
+cd ${RSTPATH}/tables/superdarn/hdw/ 
+git checkout ${TAG_VERSION}
```

Comment:
I got an error here when I ran `make.build`: 

`error: pathspec '0.0' did not match any file(s) known to git`

So I think the script can't find the subrepo. Have I done something wrong? I get the same error with `make.build 1.0`

Reply:
do you have a .gitmodules? in your main RST directory?

Reply:
You can also check by going to the hdw directory and going git branch and see what comes up.

Reply:
It also may not work for you because of being a repo of mine... this is an example of how easy things could be to get to work not necessarily wanting to spend a lot of time troubleshooting until you actually want to consider the idea for the hdw. 

Last I heard is it is currently staying the same.